recipes/sqlobject/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "SQLObject" %}
+{% set version = "3.5.0" %}
+{% set sha256 = "5d7615c23f0ed90094b492859239cb4d40d2ac8c3fa52bc7175cf417f8393509" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```

Comment:
I think you might be able to make this `noarch: python`:

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages

Reply:
Agreed, superb!